js/extensions/cadastrapp/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from '../../../MapStore2/web/client/libs/ajax';
+
+let baseURL = '/cadastrapp';
+
+export function setBaseURL(url) {
+    baseURL = url;
+}
+
+export function getConfiguration() {
+    return axios.get(`${baseURL}/services/getConfiguration`);
+}
+export function getManifest() {
+    return axios.get("/mapfishapp/ws/addons/cadastrapp/manifest.json");
```

Comment:
Not certain this method will be used, but it feels wrongto point back at a service from mapfishapp... sure, it provides the default values (and translation strings) for the addon, but what if mapfishapp isnt installed anymore ? what if mfapp isnt at `/mapfishapp` ?

Reply:
Yes, correct.
I'll remove this entry point from the API and I'll use the information from this service to map the configurations options and localization strings of the new plugin.